app/src/main/java/org/mozilla/fenix/settings/creditcards/controller/CreditCardEditorController.kt

```diff
@@ -52,36 +55,57 @@ interface CreditCardEditorController {
  * @param ioDispatcher [CoroutineDispatcher] used for executing async tasks. Defaults to [Dispatchers.IO].
  */
 class DefaultCreditCardEditorController(
+    context: Context,
     private val storage: AutofillCreditCardsAddressesStorage,
     private val lifecycleScope: CoroutineScope,
     private val navController: NavController,
     private val ioDispatcher: CoroutineDispatcher = Dispatchers.IO
 ) : CreditCardEditorController {
 
+    private val metrics = context.components.analytics.metrics
+
     override fun handleCancelButtonClicked() {
         navController.popBackStack()
     }
 
+    /**
+     * Deletes an existing credit card.
+     *
+     * @param [guid] the id of the credit card to delete.
```

Comment:
There's actually no instance of us using this @param [] formatting https://searchfox.org/mozilla-mobile/search?q=%40param+%5B&path=

```suggestion
     * @param guid the id of the credit card to delete.
```